### PR TITLE
Add name option to $TextSplit And $SplitText functions

### PR DESCRIPTION
**Pull request by overskul**
## Type
- [ ] Bug Fix
- [ x] Functions: \$textSplit, $splitText
- [ ] Callbacks: \<Callback Name>
- [ ] Handlers: \<Handler Type>
- [ ] Others: \______

Dependencies (Third Party Modules) needed: <Name | Github Link> (Maximum: 20MB~ size)

Want a credit? Discord tag or other social media link: <DiscordTag | OtherSocialMediaLink>

Referenced Issue: #NaN (Answer for an issue, if any)

## Description
Add `name` option to `$textSplit` and `$splitText` to make more than one in cmd, and that could be useful and not use `$advancedTextSplit` many times in code which make it not readable if have more than one split.

### CODE REVIEW

**Comment by Faf4a** [comment]
What about other `$textSplit` related functions? Like `$getTextSplitLength`.

**Comment by Leref** [comment]
This may be consider breaking change since many users need update this function which is quite a used function.

**Comment by Cyberghxst** [comment]
Not really, you can add a optional last field to reference the name.

**Comment by Leref** [comment]
All of the name options should be first, however right now it won't be.
It can be last, update this PR accordingly.

**Comment by overskul** [comment]
@Leref done

File: src/functions/util/getTextSplitLength.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 module.exports = d => {
     const {code} = d.util.aoiFunc(d);
+    const inside = d.unpack();
+
```
Comment (Faf4a) [inline src/functions/util/getTextSplitLength.js:4]: This method should be your last resort, use:
```js
const data = d.util.aoiFunc(d);
// (preferred) if (data.err) return d.error(data.err);

const [...] = data.inside.splits;
```

File: src/functions/util/concatTextSplit.js [hunk 2]
```diff
@@ -4,9 +4,9 @@ module.exports = async d => {
     const err = d.inside(inside);
     if (err) return d.error(err);
 
-    let [...text] = inside.splits;
+    let [...text, name = 'main'] = inside.splits;
     text = text.map(x => x.addBrackets());
```
Comment (Faf4a) [inline src/functions/util/concatTextSplit.js:8]: Same here, even if it used this method before, please update it to <data>.inside.splits

**Comment by Faf4a** [comment]
Everything besides that, lgtm

**Comment by Leref** [comment]
Agreed, it needs to use new methods

**Comment by Leref** [comment]
No changes, closing until resolved.

**Pull request closed without merging**